README.md

```diff
@@ -1 +1,43 @@
-# ziti-tunneler-sdk
\ No newline at end of file
+# API Notes
+
+The ziti tunneler SDK dispatches layer 3 packets to ziti service connections.
+Using the SDK, a tunneler application only needs to proxy layer 3
+packets from a virtual NIC (typically a tun interface)
+
+typedef void (*to_client_cb)(uint8_t *packet, size_t len)
+
+- `NF_tunnel_init(to_client_cb)`
+
+
+implement the following callbacks:
+
+- `on_service_add(const char *, intercept_address)`
+  `on_service_del(const char *, intercept_address)`
+
+  These functions are called when a service is added to an appwan.
+  Implementing applications should establish routes to the virtual
+  NIC for the intercept address of the added service.
+
+
+
+- `to_ziti(uint8_t *packet, size_t len)`
+
+  Implementing applications must call this function when a packet is
+  read from the virtual NIC.
+
+
+- `to_client(uint8_t *packet, size_t len)`
```

Comment:
is this the same as to_client_cb?

Reply:
This entire readme is stale, will update when the dust settles.